src/test/java/org/gridsuite/voltageinit/utils/assertions/ReflectiveRepresentation.java

```diff
@@ -0,0 +1,30 @@
+package org.gridsuite.voltageinit.utils.assertions;
```

Comment:
Not used ? To remove

Reply:
used when test fail, initialized through Java SPI system.

Reply:
To be removed